lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -39,7 +39,7 @@ def run_loop
             logger.debug("#{self.class}##{__method__}") { e }
             driver.emit(:close, WebSocket::Driver::CloseEvent.new(1001, 'server closed connection')) unless @closing
           ensure
-            terminate
+            terminate if ::Celluloid::Actor.current.alive?
```

Comment:
- When I use `alive?` and I keep getting 
```ruby
NoMethodError: undefined method `alive?' for #<Slack::RealTime::Concurrency::Celluloid::Socket:0x007fc8173f83d8>
```
For some strange reason `alive?` is resolved against `Slack::RealTime::Concurrency::Celluloid::Socket` instead of `Celluloid::Proxy::Cell(Slack::RealTime::Concurrency::Celluloid::Socket)`

- `Actor.current` refers to the current actor and it is thread safe.

https://github.com/celluloid/celluloid/blob/master/lib/celluloid/actor.rb#L17